### DISCUSSION TITLE
Issue with the payment

### DISCUSSION THREAD

**Discussion by mateusabelli** [community/community#145749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello there!

I am Mateus and I'm having an issue with the GitHub Sponsors payment. It has been over 1 year and the values that are shown on my Stripe Express account weren't deposited on my bank account.

Thank you in advance.

**Comment by mecodeatlas** [comment]
Hi @mateusabelli! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them. I also recommend checking out the [GitHub Sponsors documentation](https://docs.github.com/en/sponsors), which provides detailed answers to your main questions.
